lib/shopify-cli/forms/create_app.rb

```diff
@@ -0,0 +1,78 @@
+require 'shopify_cli'
+require 'uri'
+
+module ShopifyCli
+  module Forms
+    class CreateApp < Form
+      positional_arguments :name
+      flag_arguments :title, :type, :app_url, :organization_id, :shop_domain, :host
+
+      def ask
+        self.title ||= fallback_title
+        self.type = ask_type
+        self.app_url = ask_app_url
+        self.organization_id ||= organization["id"].to_i
+        self.shop_domain ||= ask_shop_domain
+        self.host ||= Tasks::Tunnel.call(ctx)
+      end
+
+      private
+
+      def fallback_title
+        name.gsub(/(.)([A-Z])/, '\1 \2') # change camelcase to title
+          .gsub(/(-|_)/, ' ') # change snakecase to title
+          .capitalize
+      end
+
+      def ask_app_url
+        while app_url !~ /\A#{URI::DEFAULT_PARSER.regexp[:ABS_URI]}\z/
+          ctx.puts('Invalid URL') unless app_url.nil?
+          self.app_url = CLI::UI::Prompt.ask('Application Url')
```

Comment:
I have a feeling we should make this more friendly? 'What is your Application URL?'